viewer/pom.xml

```diff
@@ -224,19 +236,42 @@
                         <goals>
                             <goal>npm</goal>
                         </goals>
-                        <phase>compile</phase>
+                        <phase>process-resources</phase>
                         <configuration>
                             <arguments>run build-prod</arguments>
                         </configuration>
                     </execution>
                 </executions>
                 <configuration>
-                    <workingDirectory>../tailormap-components</workingDirectory>
+                    <workingDirectory>${tailormap-components.source.dir}</workingDirectory>
                     <installDirectory>${project.build.directory}/</installDirectory>
                     <nodeVersion>v13.8.0</nodeVersion>
-                    <npmVersion>6.13.6</npmVersion>
+                    <npmVersion>6.14.8</npmVersion>
                 </configuration>
             </plugin>
+            <plugin>
+                <artifactId>maven-resources-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-resources</id>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.build.directory}/${project.artifactId}-${project.version}/tailormap-components</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>${tailormap-components.dist.dir}/</directory>
+                                    <!--includes>
+                                        <include>... misschien *.js</include>
```

Comment:
@mtoonen Not sure about this, there are some font files (material icons) in here as well that are probably needed